catroid/src/org/catrobat/catroid/common/LookData.java

```diff
@@ -114,9 +114,6 @@ public Pixmap getOriginalPixmap() {
 		return originalPixmap;
 	}
 
-	public LookData() {
-	}
```

Comment:
https://github.com/Catrobat/Catroid/pull/1216#issuecomment-121588850
The check has to be excluded.